crawl-ref/source/dat/des/builder/food.des

```diff
@@ -427,8 +427,8 @@ MONS:   human ; animal skin . hand axe
     mons("sheep w:" .. sheep_weight .. " / hog w:1 / yak w:1")
   end
 }}
-ITEM:   slice of pizza q:7, bread ration q:2
-ITEM:   beef jerky q:2
+ITEM:   slice of pizza q:7, stamina potion q:2
+ITEM:   royal jelly q:2
```

Comment:
we can't do royal jelly's anymore. All food items are gone now. We can eventually replace that with a special stamina booster potion if that makes sense.